### ISSUE TITLE
Deprecate "bridge" config entry for qa

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#385]
Originally reported of Google Code with ID 329.

```
Currently qa reads the "bridge" config entry and passes it as --master-netdev *and* --bridge to master init.

1) --bridge doesn't exist anymore, it's --nic-parameters=link:NAME
2) there's a reason the two are distinct, and we should keep them as distinct.

So we should have two separate parameters and use those.

Thanks,

Guido

```
Originally added on 2012-12-06 12:46:04 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-12-06 12:47:52 +0000 UTC.

Changed State: New
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
This is fixed in:
558e2e8 QA: Support master-netdev and default NIC parameters
cbf5114 QA: Do not pass "--bridge" to "gnt-cluster init"

```
Originally added on 2013-03-01 19:30:28 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
